client/lib/common/src/components/pure/Brush.jsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react'
+
+import { Brush } from 'react-d3-components';
```

Comment:
import { Brush as D3Brush } from 'react-d3-components'

Then you don't need the `ZBrush` :wink: